### ISSUE TITLE
Some questions about fast system calls

### ISSUE THREAD

**Issue by a74731248** [ssrg-vt/hermitux#32]
as mention in the end of [wiki](https://github.com/ssrg-vt/hermitux/wiki/Fast-system-calls-in-dynamically-linked-programs) 
> Next one can use the test-dyn target of the template Makefile to launch dynamically compiled binaries with our transformed version of Musl. Note that in the current state of things, using that template Makefile to compile static binaries (for example with make test) will succeed and create static binaries with that same unikernelized libc: they will have fast syscalls but will not work on Linux.

1. does it mean when I try HermiTux within container, and compile bunaries with static/dynamic linking, that all of these binaries use fast system calls? 
2. If I want to use a dynamic linking program (already been compiled) from outside with fast system calls, what should I do? Just download it and use the command like below?
```
HERMIT_ISLE=uhyve HERMIT_TUX=1 \
	path-to-proxy \
	path-to-hermitux \
	/lib64/ld-linux-x86-64.so.2 ./dynamic-linking-program-bynary
```

**Comment by olivierpierre** [comment]
Please see this wiki page: https://github.com/ssrg-vt/hermitux/wiki/Fast-system-calls-in-dynamically-linked-programs

**Comment by a74731248** [comment]
yes, I have seen that but is just describe how to build a unikernel-aware musl libc, rather than how to run a dynamic program without recompile it.
